### PR TITLE
Add --error and --warning options

### PR DESCRIPTION
**Pull request by misteroneill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -44,6 +42,28 @@ $ vjsstandard foo.js src/**/*.js
 
 If _not_ provided, `vjsstandard` will recursively check all JavaScript source files in the current working directory.
 
+### Options
+
+#### `-e`/`--errors`
+
+Produces a report that only includes errors; not warnings.
+
```
Comment (BrandonOCasey) [inline README.md]: good idea

File: filterer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  // If neither are true, no filtering happens.
+  } else {
+    return results;
+  }
+
+  // Handle both --errors and --warnings in the same way.
+  return results.
+
+    // Filter down to only those results which have a count of the appopriate type.
+    filter(r => r[key] > 0).
```
Comment (BrandonOCasey) [inline filterer.js]: will this ever run with the return above it?

Comment (misteroneill) [inline filterer.js]: Yeah, the return value will be the result of the entire chain.

Comment (BrandonOCasey) [inline filterer.js]: oh I didn't even notice the dot there

**Comment by BrandonOCasey** [comment]
LGTM

**Pull request merged**